R/compiled.R

```diff
@@ -109,5 +144,42 @@ line_coverages <- function(source_file, matches, values, functions) {
   names(res) <- lapply(res, function(x) key(x$srcref))
 
   class(res) <- "line_coverages"
+
+  res
+}
+
+br_coverages <- function(source_file, matches, values, functions) {
+
+  src_file <- srcfilecopy(source_file, readLines(source_file))
+
+  line_lengths <- vapply(src_file$lines[as.numeric(matches$line)], nchar, numeric(1))
+
+  res <- Map(function (br, line, length, value, func) {
+    src_ref <- srcref(src_file, c(line, 1, line, length))
```

Comment:
A comment would help to understand how the srcref is imputed

Reply:
Done!